app/grandchallenge/participants/api.py

```diff
@@ -0,0 +1,68 @@
+from django.db.models import Prefetch
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import mixins
+from rest_framework.settings import api_settings
+from rest_framework.viewsets import GenericViewSet
+
+from grandchallenge.api.permissions import IsAuthenticated
+from grandchallenge.challenges.models import Challenge
+from grandchallenge.core.guardian import filter_by_permission
+from grandchallenge.participants.models import (
+    RegistrationQuestion,
+    RegistrationQuestionAnswer,
+    RegistrationRequest,
+)
+from grandchallenge.participants.renderers import (
+    RegistrationRequestCSVRenderer,
+)
+from grandchallenge.participants.serializers import (
+    RegistrationRequestSerializer,
+)
+
+
+class RegistrationRequestViewSet(mixins.ListModelMixin, GenericViewSet):
+    serializer_class = RegistrationRequestSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
This should have object based permissions, otherwise this is all public information.

Reply:
There are permission checks in the get_queryset. Currently, we don't have any `"view_registrationrequest"` permissions being assigned to the challenge.admins_groups. Should we? If so, I'll create a PR + manual call to have them added first, before doing anything that relies on them.